ToolBox/CoinApiDataConverter/CoinApiBitfinexSymbolMapper.cs

```diff
@@ -0,0 +1,99 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Brokerages;
+
+namespace QuantConnect.ToolBox.CoinApiDataConverter
+{
+    /// <summary>
+    /// Mapper of CoinApi Bitfinex symbols with more than 3 characters.
+    /// </summary>
+    /// <seealso cref="QuantConnect.Brokerages.ISymbolMapper" />
+    public class CoinApiBitfinexSymbolMapper : ISymbolMapper
+    {
+        private HashSet<Tuple<string, string>> _mappedSymbols = new HashSet<Tuple<string, string>>
```

Comment:
I believe this collection could be `static`?
Maybe to obtain O(1) lookup for both mapping operations, we could use two dictionaries, one of which is lazy initialized as an inverted version of the other, thoughts?
```
new Lazy<Dictionary<string, string>>(() => _mappedSymbols.ToDictionary(kp => kp.Value, kp => kp.Key));
```